qutebrowser/config/configfiles.py

```diff
@@ -83,7 +83,8 @@ class YamlConfig:
     def __init__(self):
         self._filename = os.path.join(standarddir.config(auto=True),
                                       'autoconfig.yml')
-        self.values = {}
+        self._values = {}
+        self._changed = None
```

Comment:
FWIW I'd prefer this to be called `self._dirty` as it's called that in other places, and when we add a `changed` signal, it's harder to mix those two up.

Also, why set it to `None` here and not `False`?

Reply:
I made it dirty. I prefer `None` to `False` here because you don't really know whether it's changed or not: it's not even loaded. It doesn't make a difference at the moment, but it might in the future.